metpy/plots/declarative.py

```diff
@@ -673,9 +673,11 @@ class Plot2D(HasTraits):
     level = Union([Int(allow_none=True, default_value=None), Instance(units.Quantity)])
     time = Instance(datetime, allow_none=True)
 
+    clabels = Bool(default_value=False)
     colormap = Unicode(allow_none=True, default_value=None)
     image_range = Union([Tuple(Int(allow_none=True), Int(allow_none=True)),
                          Instance(plt.Normalize)], default_value=(None, None))
+    colorbar = Unicode('horizontal')
```

Comment:
How about:
```suggestion
    colorbar = Unicode(default_value='horizontal', allow_none=True)
```
I'm not sure if `Unicode('horizontal')` means that it always has a value of `'horizontal'`. Also, by allowing `None`, we have a value we can use to turn it off (I'm fine with that not being the default).

Reply:
Done.